packages/server/src/service/letter/views/Wyoming.md

```diff
@@ -0,0 +1,10 @@
+{% extends "Base.md" %}
+
+{% set guidance = 'the [Wyoming Secretary of State](https://sos.wyo.gov/Elections/State/AbsenteeVoting.aspx)' %}
+
+{% block body %}
+- County: **{{county}}**
+- Affirmation: Per the requirements: I am affirming that I am eligible to vote in the election.
+- Elections: I am requesting a ballot-by-mail for the 2020 Primary and General Elections and am giving the County Recorder permission to mail my ballot to the address provided in this email.
```

Comment:
Let's just do the general election since we need to indicate political party if we're going to do the primary.  In which case, there's an env var for that.

Reply:
I think that you fixed this yourself?